app/src/main/res/layout/fragment_movie_details.xml

```diff
@@ -1,10 +1,272 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
what if screen's height is less than content height?

Reply:
Ok, I'll add scrollView